packages/dialtone-vue3/components/rich_text_editor/rich_text_editor_default.story.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <dt-rich-text-editor
-    v-model="value"
+    v-model="$attrs.value"
```

Comment:
This is not correct, the value is defined in the component data on line 43, so it should only be

```suggestion
    v-model="value"
```

we only add the `$attrs` when this is coming from storybook